test/nightly/flink_validation_passthrough_high_availability_spec.rb

```diff
@@ -120,6 +120,7 @@
       @flink_job.kafka_producer.produce(line, key: "#{key}", topic: @input_topic, headers: {batchId: @batch_id})
       @flink_job.kafka_producer.deliver_messages
       if key == 10
+        puts "KUBECTL OUTPUT: #{@request_helper.exec_command("kubectl get namespaces")[:stdout]}"
         taskmanager_pod = @request_helper.exec_command("kubectl get pods -n #{ENV['NAMESPACE']}")[:stdout].split("\n").select { |s| s.include?('taskmanager') }[0].split(' ')[0]
         @request_helper.exec_command("kubectl delete pod #{taskmanager_pod} -n #{ENV['NAMESPACE']}")
         raise "Kubernetes pod #{taskmanager_pod} not deleted" unless @request_helper.exec_command("kubectl get pods -n #{ENV['NAMESPACE']}")[:stdout].split("\n").select { |s| s.include?(taskmanager_pod) }.empty?
```

Comment:
Just curious, is there a reason why we set `@git_branch = ENV['BRANCH_NAME']` and use `@git_branch` throughout the code, but for `NAMESPACE` it's always referenced directly from the environment variable as `ENV['NAMESPACE']`?

Reply:
I actually noticed that there were a lot of instances of both ENV['TRAVIS_BRANCH'] and @travis_branch in the private repo, so I consolidated those when changing to the Git branch. I didn't realize this was the case with NAMESPACE as well. I will update this.